src/Compilers/CSharp/Portable/Lowering/AsyncRewriter/AwaitExpressionSpiller.cs

```diff
@@ -584,6 +573,15 @@ public override BoundNode VisitConditionalGoto(BoundConditionalGoto node)
             return UpdateStatement(builder, node.Update(condition, node.JumpIfTrue, node.Label), substituteTemps: true);
         }
 
+        public override BoundNode VisitSwitchDispatch(BoundSwitchDispatch node)
```

Comment:
>VisitSwitchDispatch [](start = 34, length = 19)

Consider moving to where `VisitSwitchStatement` was, so it's clear what changed.

Reply:
Recorded in [https://github.com/dotnet/roslyn/issues/25663](https://github.com/dotnet/roslyn/issues/25663)

---
In reply to: [176260485](https://github.com/dotnet/roslyn/pull/25544#discussion_r176260485) [](ancestors = 176260485)